resources/deploy_intermediate_fedora.sh

```diff
@@ -0,0 +1,114 @@
+
+mkdir -p "/usr/share/vpn-provisioning/certs"
+cp ./MicrosoftIntuneRootCertificate.cer "/usr/share/vpn-provisioning/certs/MicrosoftIntuneRootCertificate.cer"
+
+mkdir -p "/usr/share/vpn-provisioning/web"
+cp ./index.php "/usr/share/vpn-provisioning/web"
+
+# update hostname
+sed -i "s/vpn.example/${INTERMEDIATE_FQDN}/" "/etc/httpd/conf.d/${INTERMEDIATE_FQDN}.conf"
+sed -i "s/vpn.example/${INTERMEDIATE_FQDN}/" "/usr/share/vpn-provisioning/web/index.php"
+
+# update vpn name
+sed -i "s/{vpnUrl}/${VPN_URL//\//\\/}/" "/usr/share/vpn-provisioning/web/index.php"
+
+# update tenant id
+sed -i "s/{tenantId}/${TENANT_ID}/" "/usr/share/vpn-provisioning/web/index.php"
+
+# update application id
+sed -i "s/{applicationId}/${APPLICATION_ID}/" "/usr/share/vpn-provisioning/web/index.php"
+
+# update secret application token
+sed -i "s/{secretToken}/${${SECRET_TOKEN//\\/\\\\}//\//\\/}/" "/usr/share/vpn-provisioning/web/index.php"
+
+# update admin api token
+sed -i "s/{adminApiToken}/${ADMIN_API_TOKEN}/" "/usr/share/vpn-provisioning/web/index.php"
+
+###############################################################################
+# CERTBOT
+###############################################################################
+
+systemctl stop httpd
+
+certbot certonly --standalone -d "${INTERMEDIATE_FQDN}"
+
+cat << EOF > /etc/sysconfig/certbot
+PRE_HOOK="--pre-hook 'systemctl stop httpd'"
+POST_HOOK="--post-hook 'systemctl start httpd'"
+RENEW_HOOK=""
+CERTBOT_ARGS=""
+EOF
+
+sed -i "s/#SSLEngine/SSLEngine/" "/etc/httpd/conf.d/${INTERMEDIATE_FQDN}.conf"
+sed -i "s/#Redirect/Redirect/" "/etc/httpd/conf.d/${INTERMEDIATE_FQDN}.conf"
+
+sed -i "s|#SSLCertificateFile /etc/letsencrypt/live/${INTERMEDIATE_FQDN}/cert.pem|SSLCertificateFile /etc/letsencrypt/live/${INTERMEDIATE_FQDN}/cert.pem|" "/etc/httpd/conf.d/${INTERMEDIATE_FQDN}.conf"
+sed -i "s|#SSLCertificateKeyFile /etc/letsencrypt/live/${INTERMEDIATE_FQDN}/privkey.pem|SSLCertificateKeyFile /etc/letsencrypt/live/${INTERMEDIATE_FQDN}/privkey.pem|" "/etc/httpd/conf.d/${INTERMEDIATE_FQDN}.conf"
+sed -i "s|#SSLCertificateChainFile /etc/letsencrypt/live/${INTERMEDIATE_FQDN}/chain.pem|SSLCertificateChainFile /etc/letsencrypt/live/${INTERMEDIATE_FQDN}/chain.pem|" "/etc/httpd/conf.d/${INTERMEDIATE_FQDN}.conf"
+
+systemctl start httpd
+
+###############################################################################
+# CRON
+###############################################################################
+
+install -m 600 etc/vpn-provisioning "/etc/vpn-provisioning"
+
+sed -i "s/vpn.example/${INTERMEDIATE_FQDN}/" "/etc/vpn-provisioning"
+sed -i "s/{vpnUrl}/${VPN_URL//\//\\/}/" "/etc/vpn-provisioning"
+sed -i "s/{adminApiToken}/${ADMIN_API_TOKEN}/" "/etc/vpn-provisioning"
+sed -i "s/{tenantId}/${TENANT_ID}/" "/etc/vpn-provisioning"
+sed -i "s/{applicationId}/${APPLICATION_ID}/" "/etc/vpn-provisioning"
+sed -i "s/{secretToken}/${${SECRET_TOKEN//\\/\\\\}//\//\\/}/" "/etc/vpn-provisioning"
```

Comment:
Here is a nested parameter as well, please fix according to https://www.shellcheck.net/wiki/SC2299